src/WinUsb/WinUsb.cs

```diff
@@ -0,0 +1,167 @@
+        public static extern unsafe bool WinUsb_ReadPipe(SafeUsbHandle interfaceHandle, byte pipeID, void* buffer, int bufferLength, out int lengthTransferred, NativeOverlapped* overlapped);
+
+        /// <summary>
+        /// Resets the data toggle and clears the stall condition on a pipe.
+        /// </summary>
+        /// <param name="handle">
+        /// An opaque handle to the interface that contains the endpoint with which the pipe is associated.
+        /// </param>
+        /// <param name="pipeID">
+        /// The identifier (ID) of the control pipe. The PipeID parameter is an 8-bit value that consists in a 7-bit address and a direction bit.
+        /// This parameter corresponds to the bEndpointAddress field in the endpoint descriptor.
+        /// </param>
+        /// <returns>
+        /// <see cref="WinUsb_ResetPipe"/> returns <see langword="true"/> if the operation succeeds.
+        /// Otherwise, this function returns <see langword="false"/>, and the caller can retrieve the logged error by calling <see cref="Marshal.GetLastWin32Error"/>.
+        /// </returns>
+        [DllImport(nameof(WinUsb), SetLastError = true)]
+        public static extern bool WinUsb_ResetPipe(SafeUsbHandle handle, byte pipeID);
+
+        /// <summary>
+        /// Aborts all of the pending transfers for a pipe. This is a synchronous operation.
+        /// </summary>
+        /// <param name="handle">
+        /// An opaque handle to the interface that contains the endpoint with which the pipe is associated.
+        /// </param>
+        /// <param name="pipeID">
+        /// The identifier (ID) of the control pipe. The PipeID parameter is an 8-bit value that consists in a 7-bit address and a direction bit.
+        /// This parameter corresponds to the bEndpointAddress field in the endpoint descriptor.
+        /// </param>
+        /// <returns>
+        /// <see cref="WinUsb_AbortPipe"/> returns <see langword="true"/> if the operation succeeds.
+        /// Otherwise, this function returns <see langword="false"/>, and the caller can retrieve the logged error by calling <see cref="Marshal.GetLastWin32Error"/>.
+        /// </returns>
+        [DllImport(nameof(WinUsb), SetLastError = true)]
+        public static extern bool WinUsb_AbortPipe(SafeUsbHandle handle, byte pipeID);
+
+        /// <summary>
+        /// Releases all of the resources that <see cref="WinUsb_Initialize"/> allocated. This is a synchronous operation.
+        /// </summary>
+        /// <param name="handle">
+        /// An opaque handle to an interface in the selected configuration. That handle must be created by a previous call to <see cref="WinUsb_Initialize"/> or <c>WinUsb_GetAssociatedInterface</c>.
+        /// </param>
+        /// <returns>
+        /// <see cref="WinUsb_Free"/> returns <c>TRUE</c>.
+        /// </returns>
+        [DllImport(nameof(WinUsb), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
```

Comment:
I'm seeing inconsistent use of this `[return: MarshalAs]` attribute in this PR. Strictly speaking, it's not required since it's the default behavior (and we might even update our guidelines to omit it), but we should probably be consistent at least within a file.